.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: node_js
 node_js:
   - '10'
+addons:
+  apt:
+    packages:
+      # Needed for cypress
```

Comment:
why?
isnt it already working fine?

Reply:
It was, it crashed and I followed the docs and now works again (I guess there was an update)